lib/commands/context.js

```diff
@@ -2,6 +2,7 @@ import { iosCommands, IOSPerformanceLog, NATIVE_WIN, WEBVIEW_WIN } from 'appium-
 import { createRemoteDebugger, RemoteDebugger } from 'appium-remote-debugger';
 import { errors, isErrorType } from 'appium-base-driver';
 import { util, timing } from 'appium-support';
+import pluralize from 'pluralize';
```

Comment:
I'd say it makes sense to add the pluralize feature into appium-support, since it's used in many places